app/configurator/components/field.tsx

```diff
@@ -625,7 +636,10 @@ export const ChartFieldField = ({
   optional?: boolean;
   disabled?: boolean;
 }) => {
-  const fieldProps = useChartFieldField({ field });
+  const classes = useStyles();
+  const { fetchingHierarchy, ...fieldProps } = useChartFieldField({
```

Comment:
I am thinking that fetchingHierarchy is maybe too narrow as a usecase to have it as a standalone property. I would propose to have only `fetching` here.

Reply:
Done :)